ast/src/statements/assign/mod.rs

```diff
@@ -24,22 +24,38 @@ use std::fmt;
 mod assignee;
 pub use assignee::*;
 
+/// The assignment operator.
 #[derive(Clone, Copy, PartialEq, Eq, Serialize, Deserialize, Debug)]
 pub enum AssignOperation {
+    /// Plain assignment, `=`.
     Assign,
+    /// Add-assignment, `+=`.
     Add,
+    /// Subtracting assignment, `-=`.
     Sub,
+    /// Multiplicating assignment, `*=`.
     Mul,
+    /// Divising-assignment, `/=`.
     Div,
+    /// Exponentating assignment `**=`.
     Pow,
+    /// Logical or assignment.
```

Comment:
Logical conjoining assignment? (Similar for others.) Mmhh, maybe the '-ing' form is not that great and we could instead describe them as 'Assignment combined with <operation>.'?